<launch>
    <param name="/RobotTypeName" value="Minibot"/>
    <param name="/simulation_active" value="true"/>

    <rosparam file="$(find bitbots_motion)/config/motion.yaml" command="load"/>

    <node name="motion" pkg="bitbots_motion" type="motion.py" args="" respawn="false"/>
</launch>